### DISCUSSION TITLE
b23.tv短链接支持

### DISCUSSION THREAD

**Discussion by Q16KBreak** [the1812/Bilibili-Evolved#2608] [功能建议 / Ideas]
相关资料 https://www.bilibili.com/read/cv6395726

**Comment by Geeyun-JY3** [comment]
这是希望能将站内链接转成短链接，类似于[这个网站](https://www.bilitools.top/t/4/)的功能？

**Comment by the1812** [comment]
好奇短链接的意义在哪... 不还是要复制?

**Reply by Geeyun-JY3** [reply]
可能就在于……**短**

**Reply by Q16KBreak** [reply]
短。。。有些链接 比如动态这种实在太长。。。

**Reply by ChaXxl** [reply]
短啊 ![fff](https://tva3.sinaimg.cn/large/ceeb653ely8gtf78e4a8dg202a02aweo.gif)

**Comment by liveskai** [comment]
> 相关资料 https://www.bilibili.com/read/cv6395726

![image](https://github.com/the1812/Bilibili-Evolved/assets/54350655/59d09105-b4c7-40c5-8b42-b24371f8a394)

**Reply by Geeyun-JY3** [reply]
这只是楼主给的参考资料。做不做另说。
